pojo/src/test/java/com/spotify/hamcrest/pojo/IsPojoTest.java

```diff
@@ -112,18 +112,39 @@ public void testMismatchFormatting() throws Exception {
 
     assertThat(description.toString(), is(
         "SomeClass {\n"
-        + "  ...\n"
         + "  baz(): SomeClass {\n"
-        + "    ...\n"
         + "    foo(): was <42>\n"
-        + "    ...\n"
```

Comment:
Yeah exactly. The ellipsis should only occur between mismatched fields if there are matched fields in-between